server/database/schema.sql

```diff
@@ -1,12 +1,64 @@
-create table user (
-  id int unsigned primary key auto_increment not null,
-  email varchar(255) not null unique,
-  password varchar(255) not null
+-- Vidéo
+CREATE TABLE video (
+  id INT UNSIGNED PRIMARY KEY AUTO_INCREMENT NOT NULL,
+  name VARCHAR(80) NOT NULL,
+  description TEXT,
+  date DATE NOT NULL,
+  is_premium BOOLEAN NOT NULL,
```

Comment:
you can set a default values here and next fields

Reply:
I added default values for booleans (which I assume is what you meant ?)